cfme/tests/networks/test_sdn_inventory_collection.py

```diff
@@ -1,30 +1,92 @@
 import pytest
+
 from cfme.cloud.provider.azure import AzureProvider
 from cfme.cloud.provider.ec2 import EC2Provider
-from cfme.cloud.provider.openstack import OpenStackProvider
-from cfme.networks.cloud_network import CloudNetwork
-from cfme.utils.appliance.implementations.ui import navigate_to
-
+from cfme.cloud.provider.gce import GCEProvider
+from cfme.utils.blockers import BZ
 
 pytestmark = [
-    pytest.mark.usefixtures('setup_provider'),
-    pytest.mark.provider([EC2Provider, AzureProvider, OpenStackProvider],
-                         scope='module'),
+    pytest.mark.usefixtures("setup_provider_modscope"),
+    pytest.mark.provider([EC2Provider, AzureProvider, GCEProvider], scope='module')
 ]
 
 
-@pytest.mark.tier(1)
-@pytest.mark.uncollect
-def test_sdn_inventory_subnets(provider):
-    """ Test to compare mgmt system and ui list of networks
-    Note: Those providers don't have list_network() method implemented yet
-    """
-    network_names = provider.mgmt.list_network()
-    for network_name in network_names:
-        temp_network = CloudNetwork(name=network_name)
-        view = navigate_to(temp_network, 'Details')
-        net_name = view.entities.properties.get_text_of('Name')
-        assert net_name == network_name
-
-    provider.delete_if_exists(cancel=False)
-    provider.wait_for_delete()
+def test_sdn_api_inventory_networks(provider, soft_assert):
+    prov_networks = provider.mgmt.list_network()
+    cfme_networks = [nt.name for nt in provider.appliance.collections.cloud_networks.all()]
+    prov_networks.sort()
+    cfme_networks.sort()
```

Comment:
**QUESTION**
Here and further.
What's the point to sort these lists? If it's needed for comparison then consider using `set()` instead.

Reply:
replace by sorted